test/powershell/Host/PSVersionTable.Tests.ps1

```diff
@@ -15,20 +25,29 @@ Describe "PSVersionTable" -Tags "CI" {
        $PSVersionTable.ContainsKey("OS")                        | Should Be True
 
     }
-    It "GitCommitId property should not contain an error" {
-       $PSVersionTable.GitCommitId | Should not match "powershell.version"
+
+    It "PSVersion property" {
+       $PSVersionTable.PSVersion | Should BeOfType "System.Management.Automation.SemanticVersion"
+       $PSVersionTable.PSVersion | Should BeExactly $expectedVersion
+       $PSVersionTable.PSVersion.Major | Should Be 6
+    }
+
+    It "GitCommitId property" {
+       $PSVersionTable.GitCommitId | Should BeOfType "System.String"
+       [regex]::IsMatch($PSVersionTable.GitCommitId, "^v(\d+\.\d+\.\d+)-(.+)-(\d+)-g(.+)") | Should Be $true
```

Comment:
Two things:
1. When it's building from a release tag, `$PSVersionTable.GitCommitId` will be like `v6.0.0-beta.7` and this match operation will fail.
2. It's better to use `... | Should Match ...`